pyaedt/edb_core/hfss.py

```diff
@@ -441,19 +441,77 @@ def create_coax_port_on_component(self, ref_des_list, net_list):
         if not isinstance(net_list, list):
             net_list = [net_list]
         for ref in ref_des_list:
-            for pinname, pin in self._pedb.core_components.components[ref].pins.items():
-                if pin.net_name in net_list and pin.pin.IsLayoutPin():
-                    port_name = "{}_{}_{}".format(ref, pin.net_name, pin.pin.GetName())
-                    if is_ironpython:
-                        res, fromLayer_pos, toLayer_pos = pin.pin.GetLayerRange()  # pragma: no cover
-                    else:
-                        res, fromLayer_pos, toLayer_pos = pin.pin.GetLayerRange(None, None)
-                    if self._edb.Cell.Terminal.PadstackInstanceTerminal.Create(
-                        self._active_layout, pin.pin.GetNet(), port_name, pin.pin, toLayer_pos
-                    ):
-                        coax.append(port_name)
+            for net_name in net_list:
+                port_name = self.create_coax_port_on_component_per_net(ref, net_name)
+                coax.append(port_name)
         return coax
 
+    @pyaedt_function_handler()
+    def create_coax_port_on_component_per_pin(self, reference_designator, pin_number, port_name=""):
+        """Create a coaxial port on a component per pin.
+
+        Parameters
+        ----------
+        reference_designator : str
+            Reference designator of the component.
+        pin_number : str
+            Pin number.
+        port_name : str, optional
+            Name of the net. If not specified, a name will be automatically assigned.
+        Returns
+        -------
```

Comment:
```suggestion
        port_name : str, optional
            Name of the net. The default is ``""``, in which case a name is automatically assigned.
            
        Returns
        -------
        Port name when successful; ``False`` otherwise.
```